### DISCUSSION TITLE
Renovate self-hosted docker real always on?

### DISCUSSION THREAD

**Discussion by zisuu** [renovatebot/renovate#20197] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34

### If you're self-hosting Renovate, select which platform you are using.

None

### If you're self-hosting Renovate, tell us what version of the platform you run.

platform: docker

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Is there a way to have a docker or k8s (selfhosted) Renovate Bot running real always on, always like 7dx24h without the requirement of an external trigger/scheduling thing.

In my opinion, if renovate bot is acting as service it really should handle itself and not relay on some external container scheduling/trigger things like Cron or Pipeline trigger. I would prefer to have a Renovate Bot which is running 7dx24h and does not just stop after a "onetime shot".

Any ideas how I could achieve this?

### What you tried so far.

Running hourly with crontab which requires an external trigger which I want to avoid.

### Relevant debug logs

logs are irrelevant for this question

**Comment by rarkins** [comment, marked as answer]
https://github.com/mend/renovate-on-prem

**Reply by zisuu** [reply]
thanks a lot for the link, did not know that there is another Renovate edition available. I try to figure out the difference of those Renovate editions: https://hub.docker.com/r/whitesource/renovate and https://hub.docker.com/r/renovate/renovate but couldn't find an explanation. Is the only difference the License? Can someone help me to figure out what's the difference between those two regarding functionalities/features/limitations? Are they 1:1 the same software?

**Reply by rarkins** [reply]
https://github.com/mend/renovate-on-prem/blob/main/docs/overview.md
